src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -90,13 +89,14 @@ const getEmptyCoreStats = (): CompletionStats =>
 // This function is referenced in the recalculate-core-completion migration file.
 // Any code updates should validate that migration is unaffected.
 export const calcDonorCoreEntityStats = (
-  donor: Donor,
+  donor: Donor | DeepReadonly<Donor>,
   forceFlags: ForceRecaculateFlags, // used to control recalculate under certain conditions
 ) => {
-  const updatedDonor = cloneDeep(donor);
+  // We have to be able to modify core completion, this creates a duplicate w/ readonly removed
+  const updatedDonor = JSON.parse(JSON.stringify(donor)) as Donor;
```

Comment:
same comment here. this is just fixing the type complaints because you're creating a new object with stringify and then making it usable with parse, then telling TS its a Donor. the same outcome would be just creating a new object ref. i'm going to check out the branch / message you on slack to see exactly what the intent is here.